.codesandbox/ci.json

```
@@ -0,0 +1,9 @@
+{
+  "packages": ["packages/react", "packages/react-dom"],
+  "buildCommand": "build react/index,react-dom/index",
```

```suggestion
  "buildCommand": "build --type=UMD,NODE react/index,react-dom/index",
```

The other bundles won't be published anyway due to the publishDirectory.

---

Don't need UMD, though